### PR TITLE
doc: add doc of KubeFATE CLI user guide

### PR DESCRIPTION
**Pull request by owlet42**
Signed-off-by: ChenLong Ma <owlet42@126.com>

### CODE REVIEW

File: docs/KubeFATE_command_line_user_guide.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+```
+
+Deploy KubeFATE service on Kubernetes
+
+```
+kubectl apply -f ./rbac-config.yaml
+kubectl apply -f ./kubefate.yaml
+```
+
+*A more detailed deployment process is here([deploy KubeFATE in Kubernetes](https://github.com/FederatedAI/KubeFATE/tree/master/k8s-deploy#initial-a-new-fate-deployment)).*
```
Comment (LaynePeng) [inline docs/KubeFATE_command_line_user_guide.md:27]: Should add a note about `rbac-config.yaml` just a reference access setting.

File: docs/KubeFATE_command_line_user_guide.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
+# KubeFATE Command Line User Guide
+
+### What is KubeFATE command line
+
+KubeFATE command line is a command line tool that connects KubeFATE  service to deploy FATE on Kubernetes.
+
+### Install KubeFATE command line
+
+Before using kubefate command line, you need to deploy KubeFATE  service.
```
Comment (LaynePeng) [inline docs/KubeFATE_command_line_user_guide.md:9]: kubefate -> `kubefate`. Make sure we have cleared different objects mentioned in document:
1. Project name, be consistent with the spelling: KubeFATE;
2. Command line, make sure all wrap as code block: `kubefate`

File: docs/KubeFATE_command_line_user_guide.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+```
+kubectl apply -f ./rbac-config.yaml
+kubectl apply -f ./kubefate.yaml
+```
+
+*A more detailed deployment process is here([deploy KubeFATE in Kubernetes](https://github.com/FederatedAI/KubeFATE/tree/master/k8s-deploy#initial-a-new-fate-deployment)).*
+
+#### Install command line
+
+Kubefate command line is developed by go and can run easily on Linux, Mac OS and windows.
```
Comment (LaynePeng) [inline docs/KubeFATE_command_line_user_guide.md:31]: Same as first comment: Kubefate -> KubeFATE. 
And windows -> Windows

**Review by LaynePeng — CHANGES_REQUESTED** [review]

File: docs/KubeFATE_command_line_user_guide.md (hunk 1 above)
Comment (owlet42) [reply docs/KubeFATE_command_line_user_guide.md:27]: [deploy KubeFATE in Kubernetes](https://github.com/FederatedAI/KubeFATE/tree/master/k8s-deploy#initial-a-new-fate-deployment) This link contains deployment details, so I won’t focus on it here.

**Review by LaynePeng — APPROVED** [review]
LGTM

**Review by LaynePeng — APPROVED** [review]

**Pull request merged**